packages/types/src/ethereum/interfaces.ts

```diff
@@ -118,10 +116,14 @@ export type EthereumLog<T extends EthereumResult = EthereumResult> = {
 
 export type FlareLog<T extends FlareResult = FlareResult> = EthereumLog<T>;
 
-export type EthereumBlockWrapper = BlockWrapper<
-  EthereumBlock,
-  EthereumTransaction,
-  EthereumLog,
-  EthereumTransactionFilter,
-  EthereumLogFilter
->;
+export type LightEthereumLog<T extends EthereumResult = EthereumResult> = Omit<
```

Comment:
I don't know, does `EthereumLogLite` sound better?

Reply:
To me `LightEthereumLog` makes more sense, it follows the naming for substrate too